fabric/src/generated/resources/data/c/tags/blocks/material/netherrack.json

```diff
@@ -0,0 +1,7 @@
+{
+  "replace": false,
+  "values": [
+    "#consistency_plus:material/netherrack",
+    "minecraft:netherrack"
```

Comment:
"minecraft:nether_bricks" and "minecraft:chiseled_nether_bricks" are also part of this set